python/molequeue/utils.py

```diff
@@ -46,12 +45,9 @@ def json_to_queues(json):
   def object_to_json_params(job):
     params = {}
 
-    for key, value in job.__dict__.iteritems():
+    for key, value in job.__dict__.items():
       field = underscore_to_camelcase(key)
 
-      if type(value) == types.InstanceType:
-        value = JsonRpc.object_to_json_params(value)
```

Comment:
`types.InstanceType` is removed in python3 but I am not confident this modification is OK